nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -59,7 +59,15 @@
 
   <itemizedlist>
    <listitem>
-    <para />
+    <para>
+        Grafana is now built without support for phantomjs by default. Phantomjs support has been deprecated in grafana <link>https://grafana.com/docs/grafana/latest/guides/whats-new-in-v6-4/</link> and the <literal>phantomjs</literal> project is currently unmaintained.
+        It can still be enabled by providing <literal>phantomJsSupport = true</literal> to the package instanciation.
+        For example in nixos: <literal>
```

Comment:
Please use `programlisting` here. An example can be found here: https://github.com/NixOS/nixpkgs/blob/release-19.09/doc/release-notes.xml#L147

Additionally, the manual build is failing for me. Can you please take a look at the issue? :)

Reply:
I changed the tag as requested.

I built it successfully 2 days ago, but now I get an error as well:

```
nixos-manual-combined> manual-combined.xml:78520: element para: Relax-NG validity error : Did not expect element para there
nixos-manual-combined> 78516     <title>Release Notes</title>
nixos-manual-combined> 78517     <para>
nixos-manual-combined> 78518      This section lists the release notes for each stable version of NixOS and
nixos-manual-combined> 78519      current unstable revision.
nixos-manual-combined> 78520     </para>
nixos-manual-combined> 78521     <section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0" xml:id="sec-release-20.09">
nixos-manual-combined> 78522     <title>Release 20.09 (&#x201C;Nightingale&#x201D;, 2020.09/??)</title>
nixos-manual-combined> manual-combined.xml:78522: element section: Relax-NG validity error : Did not expect element section there
nixos-manual-combined> 78518      This section lists the release notes for each stable version of NixOS and
nixos-manual-combined> 78519      current unstable revision.
nixos-manual-combined> 78520     </para>
nixos-manual-combined> 78521     <section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0" xml:id="sec-release-20.09">
nixos-manual-combined> 78522     <title>Release 20.09 (&#x201C;Nightingale&#x201D;, 2020.09/??)</title>
nixos-manual-combined> 78523
nixos-manual-combined> 78524     <section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0" xml:id="sec-release-20.09-highlights">
nixos-manual-combined> manual-combined.xml:78522: element section: Relax-NG validity error : Element appendix has extra content: section
nixos-manual-combined> 78518      This section lists the release notes for each stable version of NixOS and
nixos-manual-combined> 78519      current unstable revision.
nixos-manual-combined> 78520     </para>
nixos-manual-combined> 78521     <section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0" xml:id="sec-release-20.09">
nixos-manual-combined> 78522     <title>Release 20.09 (&#x201C;Nightingale&#x201D;, 2020.09/??)</title>
nixos-manual-combined> 78523
nixos-manual-combined> 78524     <section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0" xml:id="sec-release-20.09-highlights">
nixos-manual-combined> manual-combined.xml:3: element info: Relax-NG validity error : Element book has extra content: info
nixos-manual-combined> 1        <?xml version="1.0"?>
nixos-manual-combined> 2        <book xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0" xml:id="book-nixos-manual">
nixos-manual-combined> 3         <info>
nixos-manual-combined> 4          <title>NixOS Manual</title>
nixos-manual-combined> 5          <subtitle>Version 20.09
nixos-manual-combined> 6          </subtitle>
nixos-manual-combined> 7         </info>
nixos-manual-combined> manual-combined.xml fails to validate
builder for '/nix/store/ybmllkl71l9mb4k7bg3fnn6h89zhpacp-nixos-manual-combined.drv' failed with exit code 3; last 10 log lines:
  manual-combined.xml:3: element info: Relax-NG validity error : Element book has extra content: info
       1  <?xml version="1.0"?>
       2  <book xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0" xml:id="book-nixos-manual">
       3     <info>
       4          <title>NixOS Manual</title>
       5    <subtitle>Version 20.09
       6        </subtitle>
       7   </info>
  
  manual-combined.xml fails to validate
cannot build derivation '/nix/store/05j8s628xvhr3mpxqycmrsx40gjdihmq-nixos-manual-html.drv': 1 dependencies couldn't be built
[0 built (1 failed)]
error: build of '/nix/store/05j8s628xvhr3mpxqycmrsx40gjdihmq-nixos-manual-html.drv' failed
```

Except I didn't change anything on the tags on which those error occur, so I don't thing my change broke the build. Maybe a change in the xml schemas ?

Reply:
So the issue was a missing `xlink:href` in your `link`-tag. Rebased onto latest master and fixed the release-notes nwo :)